Services/Component/classes/class.ilObjComponentSettingsGUI.php

```diff
@@ -517,17 +502,28 @@ function updatePlugin()
 		$pl = ilPlugin::getPluginObject($_GET["ctype"], $_GET["cname"],
 			$_GET["slot_id"], $_GET["pname"]);
 
-		$result = $pl->update();
-		
+		$this->update($pl);
+	}
+
+	/**
+	 * Execute update and ctrl reload
+	 *
+	 * @param ilPlugin 	$plugin
+	 *
+	 * @return null
```

Comment:
With PHP7.1 there will be `void` as a valid return-type (http://php.net/manual/de/migration71.new-features.php). I thinks thats why PHPStorm already starts to auto-add it.